baybe/serialization/core.py

```diff
@@ -20,6 +21,8 @@
 converter = cattrs.Converter()
 """The default converter for (de-)serializing BayBE-related objects."""
 
+configure_union_passthrough(bool | int | float | str, converter)
```

Comment:
Yes, because otherwise you'd need to register hooks for all sorts of union attributes like when you have `x: int | float`, where cattrs would complain. I had one such case in existing code before I got to know about the union strategy. It's related to this PR because it would have required to set up more such unnecessary union hooks for the new classes. So now that I know that the passthrough exists, I solved the root cause of the problem, avoiding new boilerplate code

Reply:
Ok, thanks for explaining :)